bin/tag-release.sh

```diff
@@ -14,4 +14,5 @@ done
 echo "tagged $tag_value"
 if [[ "$1" == "--push" ]]; then
   git push "$moz_git_remote" "$tag_value"
+  git push "$moz_git_remote" master:prod
```

Comment:
This assumes that you've tagged the HEAD commit of master, which is what should have happened. Something is wrong if this is wrong.